framework/console/controllers/BaseMigrateController.php

```diff
@@ -622,6 +622,12 @@ public function actionCreate($name)
         }
 
         list($namespace, $className) = $this->generateClassName($name);
+        // Check that class name less then 251 (255 - strlen('.php'))
+        if (strlen($className) > 251) {
+            $className = substr($className, 0, 251);
+            $this->stdout('Length of the migration\'s class name was truncated', Console::FG_YELLOW);
```

Comment:
Its probably better to error and let the user specify a new name. Using a truncated name does not have much value imo.

Reply:
Agree.